openhands/server/session/manager.py

```diff
@@ -53,6 +53,9 @@ async def attach_to_conversation(self, sid: str) -> Conversation | None:
         await c.connect()
         return c
 
+    async def detach_from_conversation(self, conversation: Conversation):
+        conversation.disconnect()
```

Comment:
**Suggestion:** Await the `disconnect` method as it is asynchronous.

**Existing Code:**
```Python
conversation.disconnect()
```

**Improved Code:**
```Python
await conversation.disconnect()
```

**Details:**
Ensure `disconnect` is awaited since it is an asynchronous function.